### PR TITLE
add the feature to adjust exposure manually for MT9M114 camera on VIZ…

### PR DESCRIPTION
**Pull request by alan-n-n**
…N2_board

### CODE REVIEW

**Review by Raymond0225 — COMMENTED** [review]
revert changes on .cproject and lauch file

File: firmware_app/video-utilities/camera_rtvision.cpp [hunk 1]
```diff
@@ -441,7 +442,7 @@ static void Camera_RgbIrSwitch(int8_t cameraID)
         set_iox_port_pin(BOARD_CAMERA_SWITCH_GPIO, BOARD_CAMERA_SWITCH_GPIO_PIN, cameraID);
         CAMERA_DEVICE_Start(&cameraDevice[cameraID]);
 
-#if (APP_CAMERA_TYPE == APP_CAMERA_GC0308)
+#if ((APP_CAMERA_TYPE == APP_CAMERA_GC0308) || (APP_CAMERA_TYPE == APP_CAMERA_MT9M114))
```
Comment (Raymond0225) [inline firmware_app/video-utilities/camera_rtvision.cpp]: just remove this line, we have only these 2 camera type support.

File: firmware_app/video-utilities/camera_rtvision.cpp [hunk 2]
```diff
@@ -499,7 +500,7 @@ int Camera_SetDispMode(uint8_t displayMode)
 
 int Camera_SetExposureMode(uint8_t whichCamera, uint8_t mode)
 {
-#if (APP_CAMERA_TYPE == APP_CAMERA_GC0308)
+#if ((APP_CAMERA_TYPE == APP_CAMERA_GC0308) || (APP_CAMERA_TYPE == APP_CAMERA_MT9M114))
```
Comment (Raymond0225) [inline firmware_app/video-utilities/camera_rtvision.cpp]: Remove this line

**Comment by alan-n-n** [comment]
Hi, please review for my new commit

**Review by Raymond0225 — APPROVED** [review]

**Pull request merged**